test/studies/shootout/submitted/revcomp2.chpl

```diff
@@ -17,14 +17,14 @@ config const readSize = 16384;  // the chunk size to read at a time
 
 // a channel and coordination variable for writing data to stdout
 var stdoutBin = openfd(1).writer(iokind.native, locking=false,
-                                 hints=QIO_CH_ALWAYS_UNBUFFERED),
+                                 hints = ioHintSet.direct(QIO_CH_ALWAYS_UNBUFFERED)),
```

Comment:
Ugh, I'm so sorry, I forgot to look for if there were updates to the submitted shootouts.  We try to avoid doing those until they've been updated on the site itself.  We'll want to back these out and follow the guidance in that directory's README instead.

Reply:
That's my bad, I didn't see that readme. I opened a PR to fix this here: https://github.com/chapel-lang/chapel/pull/20349

Reply:
You're not the first and you likely won't be the last, it's really hard to notice